internal/batches/executor/executor_test.go

```diff
@@ -438,7 +440,7 @@ output4=integration-test-batch-change`,
 					}
 
 					task.Steps = tc.steps
-					task.Archive = repoFetcher.Checkout(task.Repository, task.Path)
+					// task.Archive = repoFetcher.Checkout(task.Repository, task.Path)
```

Comment:
Not sure if this is up for review(it's a draft but review was requested) but this seems wrong?

Reply:
Woops. Shouldn't open PRs when I'm that tired.

And yes, this is wrong, I'll remove it, and yes, it was supposed to be ready for review :)